src/algebra/order/hom/ring.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2022 Alex J. Best, Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Alex J. Best, Yaël Dillies
 -/
+import algebra.order.archimedean
```

Comment:
Did you consider adding this content in a new file, e.g. `algebra/order/hom/archimedean.lean` instead? I guess it's not a particularly heavy import to add at this point, but I would like every newly-added import to be considered carefully.

Reply:
I don't think that's worth it, because `order_ring_hom` and `order_ring_iso` are only used for #3292.